include/swift/Driver/Job.h

```diff
@@ -177,6 +185,16 @@ class CommandOutput {
   /// first primary input.
   StringRef getAdditionalOutputForType(types::ID type) const;
 
+  /// Return a vector of additional (not primary) outputs of type \p type
+  /// associated with the primary inputs. In contrast to \c
```

Comment:
Nitpick: can you put a paragraph break in here? Doxygen and Doxygen-like tools treat the first paragraph as summary and the rest as detailed information.